qa/rpc-tests/bigblocks.py

```diff
@@ -198,8 +214,22 @@ def run_test(self):
         # Time starts a little before fork activation time:
         set_node_times(self.nodes, FORK_TIME - 100)
 
-        # No supermajority
-        self.TestMineBig(False)
+        # No supermajority, vote2mb=1
+        self.TestMineBig(False, True)
+
+        # Create a block after the expiration date. This will be rejected 
+        # by the other nodes for being more than 2 hours in the future,
```

Comment:
> Is that being asserted? 

Yes, in a roundabout manner. After generating the block in the future, we do this:

        self.nodes[0].invalidateblock(b1hash)
        assert(self.sync_blocks(self.nodes))

That manually invalidates the block only on node0, and then syncs all 4 nodes. If the other 3 nodes accepted the block at first, they would never sync unless the block was manually invalidated. The fact that that manual invalidation is not necessary for nodes 1:4 indicates that they automatically invalidated the block.